target_mvd/urls.py

```diff
@@ -13,11 +13,14 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from dj_rest_auth.registration.views import ConfirmEmailView, VerifyEmailView
 from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('dj-rest-auth/registration/account-confirm-email/<str:key>/',ConfirmEmailView.as_view(),),
```

Comment:
WDYT about changing a little bit the name of the endpoints? Like `registration/account-confirm-email/<str:key>/` for example